jaxley/utils/jax_utils.py

```diff
@@ -58,7 +58,7 @@ def nested_reshape(x):
         new_shape = tuple(nested_lengths) + x.shape[1:]
         return x.reshape(new_shape)
 
-    sub_xs = jax.tree_map(nested_reshape, xs)
+    sub_xs = jax.tree.map(nested_reshape, xs)
```

Comment:
why?

Reply:
For some reason I thought there was a better reason than the deprecation, but now I'm not finding it so I reverted. We can then update all the tree_utils in Jaxley at once at some point then.